src/visualizers/widgets/LogViewer/LogViewerWidget.js

```diff
@@ -17,6 +30,7 @@ define([
             lineNumbers: this.getLineNumbers
         });
         this.setReadOnly(true);
+        this.ansiParser = AnsiParser();
```

Comment:
When the `define` is fixed above, this won't need to be invoked (technically, `AnsiParser` is the `factory` function).